### PR TITLE
Embed css via TemplateHaskell

### PR DESCRIPTION
**Pull request by 414owen**
* Updated `style.css` to reflect the current embedded css
* Embedded `style.css` via TemplateHaskell

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -226,16 +226,24 @@ label[for='toggle-all'] {
...
+  .todo-list li .destroy {
+    color: #af5b5e;
+    display: block;
+  }
 }
 
-.todo-list li:hover .destroy {
-	display: block;
+.todo-list li .destroy:after {
+	content: '×';
```
Comment (3noch) [inline style.css]: Are these changes the result of updating `style.css` from what was embedded?

Comment (414owen) [reply style.css]: Yes, the only other place `style.css` is used is in [bench.hs](https://github.com/reflex-frp/reflex-todomvc/blob/cd15a37b0e6decf42840967ce5fba6a03cf278fa/src-bin/bench.hs), and I trust these few changed bytes won't affect the benchmark

Comment (3noch) [reply style.css]: No point on trying to fudge the benchmark by faking out your css anyway :stuck_out_tongue:

Comment (Ericson2314) [reply style.css]: Hopefully https://github.com/reflex-frp/reflex-todomvc/pull/16 didn't change it!

Comment (414owen) [reply style.css]: The latest commit is a direct copy of the latest embedded string :)

**Pull request merged**